index.css

```diff
@@ -0,0 +1,309 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+.wrapper {
+  position: relative;
+  width: 100%;
+  min-height: 38rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-grow: 1;
+}
+
+/* vertical card */
+.card-component {
+  margin-left: 2rem;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+.card-margin {
+  margin: 2rem;
+}
+.card-horizontal {
+  display: flex;
+  flex-direction: row;
+  width: 55rem;
+  max-height: 31rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+}
+.card-data {
+  width: 40rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+.img-horizontal {
+  width: 20rem;
+  height: auto;
+  margin-right: 2rem;
+  margin-left: 0.5rem;
+  border-radius: 5%;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+}
+/* .card-body{
+            font-size:2rem;
+            margin:0.5rem;
+        } */
+.card-title {
+  font-size: 3rem;
+  margin: 0.4rem;
+}
+.original-price {
+  font-size: 2rem;
+}
+.price {
+  font-size: 2rem;
+}
+.discount-price {
+  font-size: 2rem;
+}
+.btn-container-horizontal {
+  margin-left: -0.3rem;
+  margin: 0.5rem;
+}
+.card-content {
+  font-size: 2rem;
+  margin-left: 0.8rem;
+  margin-top: -1.5rem;
+  margin-bottom: 1rem;
+}
+/* card with price details */
+.price-details {
+  width: 35rem;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+}
+.pricing {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  /* border-bottom: 0.1rem solid grey; */
+}
+.price-details h1 {
+  border-bottom: 0.1rem solid grey;
+  padding: 1rem;
+  margin: 1rem;
+}
+.price-details h3 {
+  margin: 0rem;
+  font-size: 1.6rem;
+}
+.price-details p {
+  margin: 1rem;
+  font-size: 1.5rem;
+  margin-right: 0.5rem;
+}
+.button-container {
+  width: 95%;
+}
+.pricing-column {
+  align-items: center;
+  margin: 1rem;
+  font-size: 1.7rem;
+  border-bottom: 0.1rem solid grey;
+}
+.pricing-amount-before-discount {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.pricing-amount-before-discount {
+  padding: 0.5rem;
+}
+.pricing-amount-before-discount p {
+  margin-right: 0.7rem;
+}
+
+/* button component */
+
+.button-container-button {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  width: 95%;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
```

Comment:
Suggestion: If you wish to have the same border-radius on all 4 corners, you can use shorthand property instead.
Use border-radius: 1rem; 
instead of border-radius: 1rem 1rem 1rem 1rem;